_episodes_rmd/05-data-structures-part2.Rmd

```diff
@@ -208,8 +224,7 @@ We can also examine individual columns of the data.frame with our `typeof` funct
 
 ```{r}
 typeof(gapminder$year)
-typeof(gapminder$lifeExp)
-typeof(gapminder$country)
```

Comment:
I think the idea with having several examples is that they show different types. Perhaps at least leave `typeof(gapminder$country)` in to show the difference between that and the year column?